### PR TITLE
Feature/add island pyinstaller

### PR DESCRIPTION
**Pull request by itaymmguardicore**
# Feature / Fixes
> Add pyinstaller scripts for island

* [X] Have you added an explanation of what your changes do and why you'd like to include them?
* [X] Have you successfully tested your changes locally?

### CODE REVIEW

File: monkey/monkey_island/monkey_island.spec [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# -*- mode: python -*-
```
Comment (ShayNehmad) [inline monkey/monkey_island/monkey_island.spec]: This file is not PEP8

File: monkey/monkey_island/monkey_island.spec [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+def get_monkey_filename():
+    return 'monkey_island.exe' if is_windows() else 'monkey_island'
+
+
+def get_exe_strip():
+    return not is_windows()
+
+
+def get_exe_icon():
+    return 'monkey_island.ico' if is_windows() else None
```
[the remark this answers is not in the document]
Comment (itaymmguardicore) [reply monkey/monkey_island/monkey_island.spec]: I'm not sure what you mean. :S

File: monkey/monkey_island/windows/run_server_py.bat [hunk 3]
```diff
@@ -0,0 +1,4 @@
+if not exist db mkdir db
+start windows\run_mongodb.bat
+start windows\run_cc.bat
```
[the remark this answers is not in the document]
Comment (itaymmguardicore) [reply monkey/monkey_island/windows/run_server_py.bat]: This is intentional, this script runs the island using python the way we used to do all the time. run_server.bat will now use run_cc_exe.bat

File: monkey/monkey_island/monkey_island.spec (hunk 1 above)
Comment (itaymmguardicore) [reply monkey/monkey_island/monkey_island.spec]: except for a few (not so) long lines it is

File: monkey/monkey_island/monkey_island.spec (hunk 2 above)
[the remark this answers is not in the document]
Comment (ShayNehmad) [reply monkey/monkey_island/monkey_island.spec]: The icon isn't actually an Island it it's just the monkey which might cause confusion 
Small thing - delete if irrelevant

File: monkey/monkey_island/windows/run_server_py.bat (hunk 3 above)
[the remark this answers is not in the document]
Comment (ShayNehmad) [reply monkey/monkey_island/windows/run_server_py.bat]: Then it's a good idea to think about documentation other then the name - I didn't understand it the first time 🤔

File: monkey/monkey_island/monkey_island.spec (hunk 2 above)
[the remark this answers is not in the document]
Comment (itaymmguardicore) [reply monkey/monkey_island/monkey_island.spec]: We don't have an island executable icon, we just have transparent and yellow background :)
For now, nothing to do about it

File: monkey/monkey_island/windows/run_server_py.bat (hunk 3 above)
[the remark this answers is not in the document]
Comment (itaymmguardicore) [reply monkey/monkey_island/windows/run_server_py.bat]: I'll add a short description in a comment at the beginning of .bat files

File: monkey/monkey_island/requirements.txt [hunk 4]
```diff
@@ -9,10 +9,12 @@ flask
 Flask-Pymongo
 Flask-Restful
 Flask-JWT
-jsonschema
+jsonschema==2.6.0
```
Comment (danielguardicore) [inline monkey/monkey_island/requirements.txt]: Why do we hard fix the version?

**Pull request merged**